Altis_Life.Altis/core/pmenu/fn_giveItem.sqf

```diff
@@ -26,12 +26,12 @@ if (isNull _unit) exitWith {ctrlShow[2002,true];};
 //A series of checks *ugh*
 if (!([_val] call TON_fnc_isnumber)) exitWith {hint localize "STR_NOTF_notNumberFormat";ctrlShow[2002,true];};
```

Comment:
Now that you opened this pull request already, why not make it a bit nicer.

- No need for the extra parenthesis around if condition
- You do not need the semicolon before curly bracket